### ISSUE TITLE
Ethical standards and reproducibility of computer models in Neurobiology

### ISSUE THREAD

**Issue by Susana465** [open-life-science/ols-6#14]
Project Lead: Susana Roman Garcia

Mentor: Siobhan Hall

Welcome to OLS-6! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.

***

## Week 1: Meet your mentor!
- [x] Meet mentor for 30 minutes 
- [x] Create an account on [GitHub](https://github.com)
- [x] Check if you have access to the HackMD notes set up for your meetings with your mentor
- [x] Prepare to meet your mentor(s) by completing a short homework provided in your shared notes
- [ ] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor 
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

## Week 2 - Cohort Call (Welcome to Open Life Science!)

- [x] Attend call or catch up via YouTube
- [x] Create an issue on the [OLS-6 GitHub repository](https://github.com/open-life-science/OLS-6/issues/new) for your OLS work and share the link to your mentor.
- [ ] Draft a brief vision statement using your goals

  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful

- [ ] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [ ] Check the [Syllabus](https://openlifesci.org/ols-6) for notes and connection info for all the cohort calls.

## Week 3: Meet your mentor!

- [ ] Meet mentor
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement
- [ ] Complete this [compare and contrast assignment](https://docs.google.com/document/d/1ukvqDRIYfvCapVMdE5hWP-0MkLNJ9T65X43O7F336Ac/edit?usp=sharing) about current and desired community interactions and value exchanges
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit?usp=sharing))
- [ ] Share a link to your Open Canvas in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [ ] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

## Week 4: Cohort Call (Tooling and roadmapping for Open projects)

- [ ] Attend call or catch up via YouTube
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

## Week 5 and later
- [ ] Meet mentor
- [ ] Create a GitHub repository for your project
- [ ] Add the link to your repository in your issue
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-6 Syllabus](https://openlifesci.org/OLS-6) for more detailed weekly notes and assignments past week 4.

## Week 6 
- [ ] Attend call or catch up via YouTube

## Week 7
- [ ] Meet mentor

## Week 8
- [ ] Attend call or catch up via YouTube

## Week 9
- [ ] Meet mentor

## Week 10
- [ ] Attend call or catch up via YouTube

## Week 11
- [ ] Meet mentor

## Week 12
- [ ] Attend call or catch up via YouTube

## Week 13
- [ ] Meet mentor

## Week 14
- [ ] Attend call or catch up via YouTube

## Week 15
- [ ] Meet mentor

**Comment by Susana465** [comment]
## VISION STATEMENT
### TLDR (Too Long Didn't Read): I work with different groups of scientists to discuss bias and reproducibility in science, so we can work towards "slower", higher quality, less oppressive research.

I work with scientists from varied research backgrounds to discuss ethics and reproducibility in creating computer models in Neurobiology. This is so that the broad science community can become aware of potential biases they continue to carry unless questioned and critiqued, so that we can move towards a more inclusive and accessible, and away from the imperialist white supremacist patriarchal culture we live in.

## Who am I working with?
I am working with scientists coming from varied backgrounds of research, typically computer and data scientists, but the spectrum is wide and varied.

## What am I doing?
I aim to discuss and bring awareness of bias and reproducibility issues in Science. In order to do so, I will discuss these topics as I go along in my PhD research, host workshops and discussions, and share outcomes with the world to use as inspiration for a more inclusive and accessible science.

## Who am I doing this for?
- For anyone who wants to get to know how to apply ethics into Scientific Research.
- For myself because I have experienced oppression, and for those who experience much worse oppression than me. Because I have experienced certain forms of oppression, I can empathize with those who have it worse than me. I want to use the privilege I have to bring awareness into spaces that might not have questioned their privilege in certain realms before.

## Why am I doing this?
-  So that other people can have access to an example of how to embed ethics when creating computer models in Neurobiology.
- I am doing this so other people can have a concrete example of how it is possible to think about reproducibility and ethics as we create computer models of biological systems.
- I am doing this to inspire change into a new world where we move away from the dominator, imperialist white supremacist patriarchal culture we live in.

**Comment by Susana465** [comment]
Link to [repository where my project lives in GitHub](https://github.com/Susana465/OLS-6-Ethical-standards-and-reproducibility-of-computer-models-in-Neurobiology)

**Comment by Susana465** [comment]
Link to landing [`README.md`](https://github.com/Susana465/OLS-6-Ethical-standards-and-reproducibility-of-computer-models-in-Neurobiology/blob/main/README.md) in my project repository

**Comment by Romullol** [comment]
Very important topic of discussion! Looking foward to learn more about this problem in your area of research. 
I liked that you highlighted some terms in your open canvas

**Comment by harisood** [comment]
Love it @Susana465! I feel as it's focused on your work it makes it v manageable and achievable within this cohort. Really interested to see what training materials you come up with 🚀

**Comment by Susana465** [comment]
# Roadmap Draft
This project aims to create a practical example of how to create ethical and reproducible research for someone doing research in Computational Neurobiology, with PhD students as the main audience target, but with the goal that it is also applicable to anyone doing research. 

To collect data on bias/quantify bias: 
 - I will explicitly mention and acknowledge bias as I write my thesis.  
 - Write explanations on how others can do this too.

To create a practical guide on thinking about ethics and reproducibility:

## Milestone 1 / Stage 1: Planning.
### Brainstorming: 
- [x] How do I quantify bias? 
- [x] How can this be included and applied to the research I am doing?
- [x] How do I translate this into something usable by others?

### Gathering information: 
- [x] Discussion with supervisors on brainstorming questions
- [x] Collate references of accounts of bias in science from books, papers
- [ ] Find a place to collate, keep track of changes, additions.
- [x] Choose which bias to focus on, which does not necessarily mean exclude/not talk about other ones at all

### Settling at the Turing
- [x] Find collaborators for this project
- [ ] Discuss details and build up project.
- [ ] Attend AI and Ethics course given by Turing Commons to build on more specific goals and tools for working around ethics and responsible research.
- [ ] Find previous similar projects (Turing Commons). 
- [ ] Learn how to construct reliable, readable, efficient research software in a collaborative environment: Attend [Research Software Engineering with Python Course](https://alan-turing-institute.github.io/rse-course/html/index.html)

## Milestone 2 / Stage 2: Prototyping.
### _Objective_: by February 2023 have an MVP ready to use.
### Writing a usable, practical guide 
- [ ] Draft Guide: how to create reproducible and ethical research during a PhD? (make more specific?)
- [ ] Clarify which data I will be using to quantify bias (which data do I use?)
- [ ] Clarify who can help me with this (who should I target, undergrads, PhDs?)
- [ ] Decide target audience of the draft guide
- [ ] Build my PhD thesis as a (fluid, not fixed) case study on how to apply reproducibility and ethics when creating computer models in Neurobiology

### Following criteria for reproducible models
- [ ] Publish model that follows and aims to pass reproducibility criteria from [BioModels Competition](https://www.ebi.ac.uk/biomodels/competition/model-of-the-year-2023)

## Milestone 3 / Stage 3: User testing
### _Objective_: March - June 2023: test MVP during practicals at the Turing/have a BSc student apply this.
- [ ] Test MVP (Minimum Viable Product) so undergrad student/other PhD students can use.
- [ ] Give a workshop on this
- [ ] Supervise a student who applies manual to their research

## Milestone 4 / Stage 4: Merging results with my PhD thesis
### _Objective_: June - Sep 2023: publish results
- [ ] Write a chapter on outcomes of this project
- [ ] Have a published Jupyter Notebook with step by step guidance
     - Project lifecycle + bias
     - Have a publishable model that is reproducible and takes into account bias.

**Comment by nilabha1512** [comment]
I have a friend, who loves Neurobiology. Seeing a project that revolves around making computer models more responsible, so to speak is an exciting thing. I will tell him to give the question of reproducible and ethical computer models some thought as begins his planning for higher education. Good luck!
